### ISSUE TITLE
Add more sorting options

### ISSUE THREAD

**Issue by mrbaseman** [WBCE/News-with-Images#98]
For instance I miss "decending order of published until date" which keeps the next upcoming events on top of a pile of news, where future ones appear somewhere at the bottom and current ones are shown first (until they vanish when the publish until date is reached).

**Comment by webbird** [comment]
This is only one more option. :D Do you have more ideas?

**Comment by mrbaseman** [comment]
Right. This is the one I would need and which I hope to contribute, when I find the time, but the idea was to take this as an opportunity to reflect if some more options would make sense.
My impression is that no fixed rule is perfect. It all depends on the dynamics, how many posts are added, how much time in advance of the intended publication date, how long they are supposed to be relevant, and even more complex: how the distribution of the posts looks like - we have a couple of regular events, that are known in advance for the whole year. They are added at the beginning of the year, with a publication date usually 2-3 weeks before the event. We have other articles about series of events which are added during the year and which should stay online for a longer period. We also have lots of individual events which are added during the year, usually a few weeks in advance of the publication date. And there are  occasional posts on short notice. 
Always the most important one should appear first. But with this mixture, it's already a challenge to decide which one is the most important one, and why.
So, I'm going to come back to this. My usual workload will remind me every other month that it could perhaps be more easy if I put more intelligence into the sorting rules. Actually I'm still missing the bright idea to solve this really satisfyingly.

**Comment by mrbaseman** [comment]
I didn't tag the release yet, but I have installed the current main branch on our website yesterday. Seems to work fine so far
